### PR TITLE
Tweak rukpak jobs:

### PR DESCRIPTION
**Pull request by ncdc**
- Add verify-deps
- Add verify-manifests
- Add verify-commits

### CODE REVIEW

**Comment by ncdc** [comment]
The verify-commits rehearsal is failing because it's using the PULL_BASE_SHA from this repo, when it should be from the target repo. This is expected in the rehearsal, and I expect an actual PR against the target repo to work.

/pj-rehearse ack

File: ci-operator/config/openshift/operator-framework-rukpak/openshift-operator-framework-rukpak-main.yaml [hunk 1]
```diff
@@ -42,6 +42,22 @@ tests:
   commands: make verify
   container:
     from: src
+- as: verify-deps
+  commands: |
+    # Verify main repo deps
+    make -f openshift/Makefile verify-deps
```
Comment (dtfranz) [inline ci-operator/config/openshift/operator-framework-rukpak/openshift-operator-framework-rukpak-main.yaml:48]: I'm not seeing this make target in the repo, is it something you're in the process of adding?

Comment (grokspawn) [reply ci-operator/config/openshift/operator-framework-rukpak/openshift-operator-framework-rukpak-main.yaml:48]: It's part of the build-machinery: https://github.com/openshift/operator-framework-rukpak/blob/main/openshift/Makefile#L3

Comment (grokspawn) [reply ci-operator/config/openshift/operator-framework-rukpak/openshift-operator-framework-rukpak-main.yaml:48]: By the transitive process of [lots-of-includes]...
https://github.com/openshift/operator-framework-rukpak/blob/main/openshift/vendor/github.com/openshift/build-machinery-go/make/targets/openshift/deps-gomod.mk

**Pull request merged**